packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -3,13 +3,18 @@ import PropTypes from 'prop-types';
 import 'terra-base/lib/baseStyles';
 import Button from 'terra-button';
 import IconCalendar from 'terra-icon/lib/icon/IconCalendar';
+import AppDelegate from 'terra-app-delegate';
```

Comment:
@tbiethman DatePicker now takes the requestFocus/releaseFocus function instead of the AppDelegate.

https://github.com/cerner/terra-core/pull/640/commits/8eddd676bf0c82b1ecd3750df7ff8e288104aab3